### DISCUSSION TITLE
Querry TEXTURE_FORMAT properties

### DISCUSSION THREAD

**Discussion by Melchiah** [DiligentGraphics/DiligentEngine#226] [Q&A] [answered]
Hello, is there a ways to find properties for given texture format ( meaning, how it can be used - as depth attachment, or color attachment.. etc., in vulkan _vkGetPhysicalDeviceFormatProperties()_  ) - Preferably, to test that format is supported for given _BIND_FLAGS_.

On my gpu(RX580), **TEX_FORMAT_D24_UNORM_S8_UINT** is reported by Diligent to be supported ( in _TextureDesc_ struct), but if its used as depth format for renderpass, Vulkan's validation layer will report error - **format (VK_FORMAT_D24_UNORM_S8_UINT) does not contain VK_FORMAT_FEATURE_DEPTH_STENCIL_ATTACHMENT_BIT** 

Thanks for answer

**Comment by TheMostDiligent** [comment, marked as answer]
Use [GetTextureFormatInfoExt](https://github.com/DiligentGraphics/DiligentCore/blob/master/Graphics/GraphicsEngine/interface/RenderDevice.h#L358)
